readme.md

```diff
@@ -22,6 +22,8 @@ This list is mainly about [CSS](https://developer.mozilla.org/docs/Web/CSS) –
 
 ## CSS References
 
+- [Learn CSS - Web.dev](https://web.dev/learn/css/) - An evergreen CSS course and reference to level up your web styling expertise.
+- [Learn Responsive Design](https://web.dev/learn/design/) - A course exploring all aspects of responsive design. Learn how to make sites that look great and work well for everyone.
```

Comment:
Please move this to the section »Fundamental concepts«. This is a pretty awesome resource.

Reply:
It was completed
Thank you